resources/styles/components/course-listings.less

```diff
@@ -49,18 +49,14 @@
     }
 
     &[data-category=physics] {
-      .tutor-course-item {
-        .tutor-physics-cover-background();
-      }
+      &[data-title=Physics]{ .tutor-course-item { .tutor-physics-cover-background(); } }
```

Comment:
There isn't one currently. The `catalog_offering_identifier` can be set to arbitrary things on the BE to reflect what the course type is, and multiple courses will share the same identifier.  My thoughts on it are that the identifier should be used to style the courses, but it shouldn't be depended on for displaying the course name.

This presents a problem for the chooser though.  It uses SVG's for displaying the course title, and we only have ones that read "Biology" or "Physics".

In this case I think it makes sense to target the title because that's what we're replacing with an image.